backend/game/cards/master_card_list.py

```diff
@@ -14,6 +14,7 @@
 from game.cards.discard_cards.multi_manipulation_cards.vast_storm import VastStorm
 
 # Victory Point Manipulation Cards
+from game.cards.discard_cards.turn_manipulation_cards.frenzy import Frenzy
```

Comment:
It's fine to put it in here but the comment makes this misleading

Reply:
Ha totally missed that thanks

Reply:
auto import got me

Reply:
Resolved, but github isn't showing the outdated tag as the change is sorta in a comment